src/SFML/Graphics/CircleShape.cpp

```diff
@@ -75,4 +75,12 @@ Vector2f CircleShape::getPoint(std::size_t index) const
     return Vector2f(m_radius, m_radius) + Vector2f(m_radius, angle);
 }
 
+////////////////////////////////////////////////////////////
+Vector2f CircleShape::getGeometricCenter() const
+{
+    if (m_pointCount == 3)
+        return (getPoint(0) + getPoint(1) + getPoint(2)) / 3.f;
```

Comment:
What's the significance of special-casing for 3 points?

Reply:
@vittorioromeo It's the one case where the geometric center is not @ `Vector2f(m_radius, m_radius)`.